matlab/lib/+IceInternal/EncapsDecoder11.m

```diff
@@ -348,6 +348,7 @@ function skipSlice(obj)
                         catch e
                             reason = sprintf('constructor failed for type %s with compact id %d', ...
                                 eval(sprintf('IceCompactId.TypeId_%d.typeId', compactId)), compactId);
+                            % TODO does this type still exist in MATLAB?
                             ex = Ice.NoValueFactoryException('', reason, reason, '');
```

Comment:
Seems this is the only occurrence, better create a separate issue to fix it.

Reply:
@bernardnormier beat me to opening an issue, and already opened a PR to fix this: #2626